### DISCUSSION TITLE
White paper between columns

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6493] [Ideas]
This idea is inspired in a Toon Boom feature:

![image](https://user-images.githubusercontent.com/74098734/114011917-a9052800-985d-11eb-8765-1300556ed084.png)

With this option, the levels behind the selected level, lower their transparency automatically, making it easier to clean the drawings.

Fix: it's "columns", not "layers", like I edit in the picture above. A little mistake.

https://user-images.githubusercontent.com/74098734/114014418-81fc2580-9860-11eb-98a0-13970f1f0571.mp4

The paper thickness let adjust the transparency, and when you uncheck this option, the transparency of the levels returns to the previous values.

**Comment by artisteacher** [comment]
I think you were right the first time - it would be layer/column. There can be multiple levels in a layer - adding paper between levels in the same layer probably wouldn't make sense.

**Comment by F-Burning** [comment]
True!

The right word is columns @artisteacher, that's what I mean.

"White paper" is only the name, the action low the opacity of all columns.

**Comment by DarrenTAnims** [comment]
The transparency option in the column header works well for doing this, I find.

**Comment by F-Burning** [comment]
> The transparency option in the column header works well for doing this, I find.

This low the opacity of all columns behind at once, depend of the column you selected.

**Comment by artisteacher** [comment]
In the timeline, you can toggle preview and camera stand visibility across all layers. Maybe opacity could be controlled the same way.

**Comment by gab3d** [comment]
I like this proposed feature, it could help make focus on something in particular, while still being able to see the faded background for reference while drawing.

But for it to be truly useful, there should be a toggle button in some place of the Visor (probably the upper section).

**Comment by F-Burning** [comment]
True! 

I would put a button like this:

![29343 (1)](https://user-images.githubusercontent.com/74098734/114164204-ef708a80-9922-11eb-9049-3dfb07ec2e8e.png)

I leave the source: 

`https://www.flaticon.com/free-icon/stationery-stacked-papers_29343`


Maybe here, on top of timeline:

![image](https://user-images.githubusercontent.com/74098734/114163819-86891280-9922-11eb-8295-2210ff936400.png)
